### PR TITLE
P4-1392 Patch a moves document relationships

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

P4-1392

### What?

- [x] Added support for patching a Move's documents via `PATCH /moves/:id`
- [x] Updates swagger relationship references for a Move to include documents

### Why?

The frontend are creating a page for viewing and changing a move's documents and they need this for associating/disassociating created documents with the move.

### Have you? (optional)

- [x] Updated API docs if necessary

### CODE REVIEW

File: spec/requests/api/v1/moves_controller_update_spec.rb [hunk 1]
```diff
@@ -127,6 +128,47 @@
           expect(response_json).to eq resource_to_json
         end
 
+        context 'when changing a moves documents', :skip_before do
+          let(:after_documents) { [after_document] }
```
Comment (willfish) [inline spec/requests/api/v1/moves_controller_update_spec.rb:132]:
```suggestion
          let(:after_documents) { [create_list(:document, 2) }
```

Comment (willfish) [inline spec/requests/api/v1/moves_controller_update_spec.rb:132]:
```suggestion
          let(:after_documents) { create_list(:document, 2) }
```

File: app/controllers/api/v1/moves_controller.rb [hunk 2]
```diff
@@ -91,15 +91,21 @@ def move_attributes
...
+        attributes.merge(documents: Document.where(id: document_ids))
       end
 
       def render_move(move, status)
         render json: move, status: status, include: MoveSerializer::INCLUDED_ATTRIBUTES
       end
 
-      def find_move
-        Move
+      def move
```
Comment (smoothcontract) [inline app/controllers/api/v1/moves_controller.rb:107]: 👍

File: .rubocop.yml [hunk 3]
```diff
@@ -69,3 +69,7 @@ Rails/TimeZone:
   Exclude:
     - "spec/**/**/*"
     - "app/services/moves/anonymiser.rb"
+
+Lint/AmbiguousBlockAssociation:
```
Comment (smoothcontract) [inline .rubocop.yml:73]: Where was rubocop flagging this?

File: spec/requests/api/v1/moves_controller_update_spec.rb [hunk 4]
```diff
@@ -333,4 +375,8 @@
       end
     end
   end
+
+  def do_patch
```
Comment (smoothcontract) [inline spec/requests/api/v1/moves_controller_update_spec.rb:379]: 👍

File: .rubocop.yml (hunk 3 above)
Comment (willfish) [reply .rubocop.yml:73]: For the `to change` and `not_to change`

**Pull request merged**